.github/workflows/backend-dev-cd.yml

```
@@ -34,57 +34,49 @@ jobs:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
 
       - name: 새로운 서버로 실행
         env:
-          SPRING_PROFILES_ACTIVE: ${{ secrets.DEV_PROFILE }}
-          SERVER_PORT: 80
-          SPRING_DATASOURCE_URL: ${{ secrets.DEV_DB_URL }}
-          SPRING_DATASOURCE_USERNAME: ${{ secrets.DEV_DB_USERNAME }}
-          SPRING_DATASOURCE_PASSWORD: ${{ secrets.DEV_DB_PASSWORD }}
-          ADMIN_TOKEN: ${{ secrets.DEV_ADMIN_TOKEN }}
+          SPRING_PROFILES_ACTIVE=${{ secrets.DEV_PROFILE }}
```

actionlint (1.7.7)

expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)